runtime/development/src/lib.rs

```diff
@@ -1291,8 +1293,75 @@ impl pallet_xcm_transactor::Config for Runtime {
 }
 
 parameter_types! {
-	pub const MaxActiveLoansPerPool: u32 = 50;
+	pub const MaxActiveLoansPerPool: u32 = 1000;
+	/// We do not need so many iterations for benchmarking,
+	/// which also increase A LOT the time of calculating weights
+	pub const MaxActiveLoansPerPoolBenchmark: u32 = production_or_benchmark!(
+		MaxActiveLoansPerPool::get(),
+		50
+	);
 	pub const MaxWriteOffPolicySize: u32 = 10;
+	pub const MaxPriceOracleMembers: u32 = 10;
+	// The benchmark distintion can be removed once
+	// <https://github.com/open-web3-stack/open-runtime-module-library/issues/920> be merged.
+	pub const MaxHasDispatchedSize: u32 = production_or_benchmark!(
+		MaxPriceOracleMembers::get(),
+		MaxActiveLoansPerPool::get()
+	);
+	pub const MaxPoolsWithExternalPrices: u32 = 50;
+	pub RootOperatorOraclePrice: AccountId = PRICE_ORACLE_PALLET_ID.into_account_truncating();
```

Comment:
Not sure how this can be used from outside without the private key. But I need to put some account as a root.